applications/openshift/master/file_groupowner_ovs_vswitchd_pid/rule.yml

```
@@ -21,13 +23,26 @@ identifiers:
 references:
     cis@ocp4: 1.1.9
 
-ocil_clause: '{{{ ocil_clause_file_group_owner(file="/run/openvswitch/ovs-vswitchd.pid", group="openvswitch") }}}'
+ocil_clause: 'Is it the case that /run/openvswitch/ovs-vswitchd.pid has group owner openvswitch or hugetlbfs?'
```

```suggestion
ocil_clause: '/run/openvswitch/ovs-vswitchd.pid has group owner openvswitch or hugetlbfs'
```

You may remove some parts of this sentence as they are added by the build system, from the `ocil_clause` [documentation](https://complianceascode.readthedocs.io/en/latest/manual/developer/06_contributing_with_content.html?highlight=ocil_clause) we have:


`
ocil_clause: This attribute contains the statement which describes how to determine whether the statement is true or false. Check out rule.yml in linux_os/guide/system/software/disk_partitioning/encrypt_partitions/: this contains a partitions do not have a type of crypto_LUKS value for ocil_clause. This clause is prefixed with the phrase Is it the case that <ocil_clause> ?.
`

---

This can be applied to the other occurrences of ocil_clause in this PR

---

Thanks! Will do

---

Done